src/components/JobConfiguration.vue

```diff
@@ -328,7 +416,19 @@ ion-list {
 
   .mobile-only {
     display: none;
-  }  
+  }
+  .job-actions {
```

Comment:
We already have an "actions" class. So job actions doesn't make sense here I think. "More actions" seems better.

Reply:
updated sir